appium/tests/specs/mock/setup/RespectsPassPhraseSessionLength.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { ekmKeySamples } from 'api-mocks/apis/ekm/ekm-endpoints';
+import { MockApi } from 'api-mocks/mock';
+import { CommonData } from 'tests/data';
+import AppiumHelper from 'tests/helpers/AppiumHelper';
+import BaseScreen from 'tests/screenobjects/base.screen';
+import {
+  SplashScreen
+} from '../../../screenobjects/all-screens';
+import MailFolderScreen from "../../../screenobjects/mail-folder.screen";
+import SetupKeyScreen from "../../../screenobjects/setup-key.screen";
+
+describe('SETUP: ', () => {
+
+  it('respects in_memory_pass_phrase_session_length and passphrase expire in that seconds', async () => {
+
+    const mockApi = new MockApi();
+    const testMessageSubject = 'Message with cc and multiple recipients and text attachment';
+    const processArgs = CommonData.mockProcessArgs;
+
+    mockApi.fesConfig = {
```

Comment:
It can be just `mockApi.fesConfig = MockApiConfig.defaultEnterpriseFesConfiguration`